app/views/pets/edit.html.erb

```diff
@@ -15,9 +15,7 @@
   <p>Description: </p>
   <%= text_field_tag :description, "#{@pet.description}" %>
   <br>
-  <p>Adoption Status: </p>
-  <%= text_field_tag :adoption_status, "#{@pet.adoption_status}" %>
-  <br>
+
```

Comment:
removed this field. Decided it would be better to have a button to trigger adoption status that would change status with an action in the pet controller